pages/1.13/security/oss/login/service-login/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: layout.pug
+navigationTitle:  Service Login
+title: Service Login
+excerpt: Logging in to DC/OS as a service
+menuWeight: 30
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# Secure service login
+
+A service logs in to DC/OS without transmitting a password by supplying a service login token in a cryptographically secure manner.
+
+Service login tokens are always generated by the service that wants to perform a service login, ideally on-demand. The idea is that they are generated for one-time usage only (for example, for a single service login procedure). Therefore, any service login token should include an expiration time.
+
+The secure service login is enabled through a service login token being signed by the private key of the service that generated it.
+
+# Login using the DC/OS CLI 
+
+**NOTE**: This demonstrates manual login for service account testing. A service login token is generated internally for the login process.
+
+**Prerequisite:**
+- [DC/OS CLI](/1.13/cli/)
+
+Using the [DC/OS CLI](/1.13/cli/) one can log in as service by specifying the `dcos-services` login provider.
+
+To test a service login using the DC/OS CLI specify the `dcos-services` login provider with the [auth login](/1.13/cli/command-reference/dcos-auth/dcos-auth-login/) command. Replace `<service-account-id>` and `<private-key-path>` with the corresponding values before executing the following command:
+
+```bash
+dcos auth login --provider=dcos-users --username=<service-account-id> --private-key=<private-key-path>
+```
+
+# Login using the IAM API
```

Comment:
Add few lines to describe `Login using the IAM API` (below line #35). 
Sub sections will follow after the description.